gulpfile.js

```diff
@@ -35,6 +35,32 @@ gulp.task('less', function () {
     .pipe(browserSync.stream());
 });
 
+var tokenWorkerConfig = {
+    devtool: 'source-map',
+    output: {
+        filename: 'tripleo_ui_token_worker.js',
+        sourceMapFilename: 'tripleo_ui_token_worker.js.map'
+    },
+    module: {
+        loaders: [
+          { test: /\.js$/, include: /src\/js\/workers/, loader: 'babel-loader' }//,
+          // { test: /disqus-thread.js$/, loader: 'babel-loader' },
```

Comment:
remove those 2 lines, it is ok to keep them in webpack.config.js for reference how webpack works.